opencog/atomspace/BackingStore.cc

```diff
@@ -46,3 +48,13 @@ bool BackingStore::ignoreAtom(Handle h) const
 	return false;
 }
 
+void BackingStore::registerWith(AtomSpace* atomspace)
+{
+	atomspace->registerBackingStore(this);
```

Comment:
again -- single-line implementations of methods.  Why? this only makes the code harder to understand, and doesn't save any typing... is this some weird work-around for public vs. private method calls??  Maybe the private methods should not be private, if this work-around is needed?